pkg/config/server_config.go

```diff
@@ -37,6 +37,9 @@ const (
 
 	// DebugConfigurationItem is the name of debug configurations
 	DebugConfigurationItem = "debug"
+
+	// DefaultTableMemoryQuota is the default memory quota for each table.
+	DefaultTableMemoryQuota = 35 * 1024 * 1024 // 35 MB
```

Comment:
Is 35 MB a reasonable value? Would it cause OOM more easily when the table amount is huge?

Reply:
@overvenus Do you have any advice on that?

Reply:
Please do not change the default setting, it may break many table (60k) scenario.

What about setting 35MB in the big_txn test?

Reply:
It's a matter of tradeoffs: after this [pr](https://github.com/pingcap/tiflow/pull/5712), the memory consumption of single table is constant. However, the test [results](https://github.com/pingcap/tiflow/pull/5712#issue-1256591663) show a lower quota value may affect throughput, so it is recommended to increase the quota of single tables to 30-40 MB.

Reply:
Maybe we could change this parameter as needed in the future.